codegen/snippet-tests/output/07_literal_types.pkl.d.ts

```diff
@@ -0,0 +1,12 @@
+/* This file was generated by `pkl-typescript` from Pkl module `07-literalTypes`. */
+/* DO NOT EDIT! */
+/* istanbul ignore file */
+/* eslint-disable */
+import * as pklTypescript from "@pkl-community/pkl-typescript"
```

Comment:
these imports arent needed now right?

Reply:
They are if we use custom types like `pklTypescript.Any`